### DISCUSSION TITLE
Mouse tracking on external html pages

### DISCUSSION THREAD

**Discussion by L0i0n0a** [jspsych/jsPsych#3276] [Q&A] [answered]
Hey everyone, does anybody know if it is possible to implement the mouse tracking extension in external HTML pages. So if a combination of the extension and the plugin is possible? Or do I have to put everything inside the stimulus in, for example button response screens?

**Comment by Shaobin-Jiang** [comment, marked as answer]
If you mean the external-html plugin and the mouse-tracking extension, yes, they can work together.

```javascript
let jsPsych = initJsPsych({extensions: [{type: jsPsychExtensionMouseTracking, params: {minimum_sample_time: 50}}]});

let trial = {
    type: jsPsychExternalHtml,
    url: './test.html',
    cont_btn: 'start',
    execute_script: true,
    force_refresh: true,
    extensions: [{type: jsPsychExtensionMouseTracking, params: {targets: ['#jspsych-content']}}],
    on_finish: function (data) {
        console.log(data);
    }
};

jsPsych.run([trial]);
```

I use the code from the documentation (see [here](https://www.jspsych.org/7.3/plugins/external-html/#loading-a-consent-form)) in my `test.html`.

**Reply by L0i0n0a** [reply]
Thank you very much! It is working now
